content/docs/1.4.0/references/settings.md

```diff
@@ -264,6 +266,20 @@ This setting allows Longhorn to automatically delete the `orphan` resource and i
 
 The value in seconds specifies the timeout of the engine to the replica(s), and the value should be between 8 to 30 seconds. The default value is 8 seconds.
 
+#### Support Bundle Manager Image
+
+Longhorn uses the support bundle manager image to generate the support bundles.
```

Comment:
Please have an example value for this, and explain there will be default one during instal or upgrade.

Reply:
Updated.